tests/test_main.py

```diff
@@ -182,6 +182,20 @@ def test_main_args(self):
             _, kwargs = MockCoder.call_args
             assert kwargs["dirty_commits"] is True
 
+    @patch('aider.main.InputOutput.confirm_ask', return_value=True)
```

Comment:
It would be better to make a proper tempfile, not just stomp a txt file into the current directory. It might overwrite a real file.

And why patch `confirm_ask`? Not sure why this test would be asking a question? In any case, you can just call main with `--yes`:

```
main(["--yes", "--message-file", message_file_path], input=DummyInput(), output=DummyOutput())
```

Reply:
Thank you!   
Earlier when removing `@patch...` it was failing, I wondered as well. Now I see! Making this PR at evening I missed earlier to remove "mock_confirm" argument in unit,test, now `@patch` is not needed! Improving python-fu as we go!

Reply:
And of course `tempfile.mktemp()` !